client/src/pages/Profile/Profile.tsx

```diff
@@ -21,7 +25,7 @@ export default function Profile(): JSX.Element {
                         <List component="nav" className={classes.menuWrapper}>
                             {listText.map((text, index) => {
                                 return(
-                                <ListItem key={index} button>
+                                <ListItem selected={index === compIndex} key={index} button onClick={() => setIndex(index)}>
```

Comment:
Please avoid the usage of indexes as keys. If there is nothing unique we can use, look into the usage of 'uuid'

Ref:  https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9